CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 ### Grafana Mimir
 
 * [CHANGE] Ingester: Increase default value of `-blocks-storage.tsdb.head-postings-for-matchers-cache-max-bytes` and `-blocks-storage.tsdb.block-postings-for-matchers-cache-max-bytes` to 100 MiB (previous default value was 10 MiB). #6764
+* [CHANGE] Ruler: the following changes apply on remote evaluation: #7004
+  * Non-erroneous responses with status code `4xx` are treated as errors.
```

Comment:
This sentence is about how things are implemented internally, but hard to understand (and not even interesting) for the final user. I think you should just mention the backoff thing (next line) and remove this line.